lib/rambulance/exceptions_app.rb

```diff
@@ -59,12 +59,16 @@ def process_action(*)
         request.GET
       rescue ActionController::BadRequest
         request.env["MALFORMED_QUERY_STRING"], request.env["QUERY_STRING"] = request.env["QUERY_STRING"], ""
+      rescue Mime::Type::InvalidMimeType
+        request.env["MALFORMED_CONTENT_TYPE"], request.env["CONTENT_TYPE"] = request.env["CONTENT_TYPE"], "text/plain"
```

Comment:
Each one of the elements should be sanitized independently to cover a case where both the query string and the content type header. You could probably do this at the beginning of this method:

```ruby
def process_action(*)
  begin
    request.content_mime_type
  rescue Mime::Type::InvalidMimeType
    request.env["MALFORMED_CONTENT_TYPE"], request.env["CONTENT_TYPE"] = request.env["CONTENT_TYPE"], "*/*"
  end

  ...
```

Reply:
cool. this seems to work! I'll update it. Thanks @yuki24